augur/refine.py

```diff
@@ -241,11 +241,27 @@ def run(args):
     if args.divergence_units=='mutations-per-site': #default
         pass
     elif args.divergence_units=='mutations':
-        L = tt.seq_len
-        for node in node_data['nodes']:
+        if not args.timetree:
+            tt.infer_ancestral_sequences()
```

Comment:
Inferring ancestral sequences like this only works (meaningfully) if the user provides an alignment file and does not request a time tree. When the user doesn't provide an alignment, `refine` mocks up a short, fake alignment of identical sequences for each node, and the "mutations" branch lengths are all equal to zero without any warning to the user.

Reply:
without an alignment, there is no way we can use mutations as units, right?